docs/developer-docs/jr-dev-rules.md

```diff
@@ -0,0 +1,61 @@
+- Not eligible to participate in collaborations
+- Not eligible to [revise](https://docs.retroachievements.org/guidelines/content/achievement-set-revisions.html) sets other than ones they've solely authored
+  - Must make a revision request in #jr-dev-forum detailing the revision plan. The Code Reviewer team will consider the request for approval
+- May not make a claim without Code Reviewer approval on a Dreamcast using the [Windows CE firmware](https://retroachievements.org/game/24833), PlayStation 2 or future 6th+ generation consoles at the discretion of the Code Review team
+  - Must make a development request in #jr-dev-forum after at least 1 set has been published
+- May not make a subset without Code Reviewer approval
+  - Must make subset request in #jr-dev-forum. If approved, subset is subject to [Subset Rules](https://docs.retroachievements.org/guidelines/content/subsets.html)
+    - May request to create a subset in addition to a primary claim if both are for the same game
+- May make a maximum of 2 sets for hacks during the program
+
+### Code Review Requests
+
+Junior Developers are required to ensure sets they submit for Code Review are of the highest quality they can make. If a set has known issues, Junior Developers must resolve them prior to requesting a Code Review.  Code Reviewers are available to answer questions and offer guidance to assist Junior Developers resolve issues.  The following rules apply to all Junior Developers intending to submit a Code Review request:
+- Must follow the ["Am I Ready for Review"](https://docs.google.com/document/d/e/2PACX-1vSYRcYpyN0W8oP9Ho0YMiUutZEs-np4JDL-Be5IfuR5oyG_92wVwgwA5BkTHywK_olmzRBjpZGehKM6/pub) guide prior to requesting a Code Review
+- May not submit sets that fail to conform to the [Writing Policy](https://docs.retroachievements.org/guidelines/content/writing-policy.html) or [Code Note Standards](https://docs.retroachievements.org/guidelines/content/code-notes.html)
+- May not submit sets that do not have original achievement badges
+- May not submit sets that do not have Rich Presence
+- May not request a code review until at least 7 days after claiming a set
+- Will describe in the Code Review request the self testing method used in order to validate set stability
+
+**Sets that fail to meet Code Review request requirements will be refused entry into the Code Review backlog until remediated**
+
+### Removal From Junior Developer Program
+
+Participation in the Junior Developer program is not an entitlement or right. Junior Developers are expected to take the program seriously, put forth significant effort, seek assistance when unable to figure something out, strive to make the highest quality set possible and maturely accept and apply Code Reviewer feedback. Junior developers may be temporarily or permanently removed from the program for the following reasons:
```

Comment:
Minor nit: I notice "Program" is capitalized in some contexts and lowercased in others. No opinion on which one is correct, just want to point it out.

Reply:
Updated as recommended.